src/mod_manager.h

```diff
@@ -26,6 +26,8 @@ const std::vector<std::pair<std::string, translation>> &get_mod_list_categories(
 const std::vector<std::pair<std::string, translation>> &get_mod_list_tabs();
 const std::map<std::string, std::string> &get_mod_list_cat_tab();
 
+mod_id get_base_id( mod_id full_id );
```

Comment:
This is a very generic function name to inject into the global namespace.

Perhaps `mod_id get_mod_base_id_from_src(mod_id src)` is better and a little clearer on the role?
`std::pair<mod_id, mod_id> get_mod_ids_from_src(mod_id src)` is another option.